src/components/top_bar/top_bar.ts

```diff
@@ -326,25 +339,34 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
       .filter((item) => !item.isVisible || item.isVisible(this.env));
   }
 
-  onClick(ev: MouseEvent) {
-    if (this.openedEl && isChildEvent(this.openedEl, ev)) {
+  onExternalClick(ev: MouseEvent) {
+    // TODO : manage the clicks events better. We need this piece of code to avoid having the event
```

Comment:
try to have max 80-100 chars per line to avoid broken lines here in github.
Same as in commit messages